src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const modifiedString = sourceString
+    .split(';')
+    .map(el => el.trim())
+    .filter(el => el !== '')
+    .map(el => el.split(':'))
+    .reduce((elA, elB) => ({
+      ...elA, [elB[0].trim()]: elB[1].trim(),
+    }), {});
```

Comment:
```suggestion
    .reduce((acc, cur) => ({
      ...acc,
      [elB[0].trim()]: elB[1].trim(),
    }), {});
```

Reply:
1 - it's not previous and current elements like in sort method callback, it's accumulator and current element
2 - start new object property at new line